packages/openid4vc/src/shared/transform.ts

```diff
@@ -78,9 +76,12 @@ export function getVerifiablePresentationFromSphereonWrapped(
     } satisfies SdJwtVc
   } else if (wrappedVerifiablePresentation.format === 'mso_mdoc') {
     if (typeof wrappedVerifiablePresentation.original !== 'string') {
```

Comment:
i think the original will ALWAYS be string if we also change the other code to always provide a string from the start. I think it's ok  as we don't need to import the library here though. Just want to be careful in depending on two mdl librs. Ideally we jus tprovide string and get back string from PEX. For the rest it should be a black box (so we're also not prone to changes to their library)